src/app/functions/server/onerep.ts

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import type { Session } from "next-auth";
+import { type Session } from "next-auth";
```

Comment:
Asking for my own education: what does this do differently?

Reply:
Pretty sure this was the linter not me doing it purposely :) There's no functional difference AFAIK just style